google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/MessageDispatcher.java

```diff
@@ -276,7 +275,13 @@ public void processReceivedMessages(List<com.google.pubsub.v1.ReceivedMessage> r
     logger.log(
         Level.FINER, "Received {0} messages at {1}", new Object[] {responseMessages.size(), now});
 
+    try {
+      flowController.reserve(responseMessages.size(), totalByteCount);
+    } catch (FlowController.FlowControlException unexpectedException) {
+      throw new IllegalStateException("Flow control unexpected exception", unexpectedException);
```

Comment:
Just use `e` for the exception variable. (Ok, I realize this existed before, it it seems odd.)

Also, why is this an `IllegalStateException`?